trunk/SUAVE/Analyses/Stability/AVL.py

```diff
@@ -445,7 +445,12 @@ def build_surrogate(self):
                 CM_sur[ii,jj]    = cm_surrogate.predict(np.array([AoA_mesh[ii,jj],mach_mesh[ii,jj]]))
                 Cm_a_sur[ii,jj]  = cm_alpha_surrogate.predict(np.array([AoA_mesh[ii,jj],mach_mesh[ii,jj]]))
                 Cn_b_sur[ii,jj]  = cn_beta_surrogate.predict(np.array([AoA_mesh[ii,jj],mach_mesh[ii,jj]]))
-                NP_sur[ii,jj]    = neutral_point_surrogate.predict(np.array([AoA_mesh[ii,jj],mach_mesh[ii,jj]]))
+                NP_sur[ii,jj]    = neutral_point_surrogate.predict(np.array([AoA_mesh[ii,jj],mach_mesh[ii,jj]]))                
```

Comment:
@mclarke2 you need to approve this change.

Reply:
There wasn't a change...The commented code below allows the sklearn to work on my own machine.  I assume my version is more up to date than the SUAVE tutorial.

Reply:
Keep those differences locally...have @mclarke2 comment on it